.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea/
+__pycache__/
```

Comment:
we have .gitignore file. edit that file to include the OmarJabari/__pycache__/ folder

Reply:
resolve the conflicte
<img width="921" alt="image" src="https://github.com/user-attachments/assets/14758d4c-fafa-4bb8-bc43-106564f61c00">